tests/utils_framework.py

```diff
@@ -553,9 +549,9 @@ def scaleDivAlpha(alpha):
 def SpectralInitializer(eps_spectral, eps_bjorck):
     warnings.warn("spectral_ and bjorck_ require n_iterations in pytorch")
```

Comment:
Is this warning still required with the new `eps_spectral` and `eps_bjorck`?

Reply:
the commit e83741e452cac91e7f6a7713f060443199b1c97f solved this issue